src/pages/SettingsPage.tsx

```diff
@@ -1,7 +1,189 @@
+} from "@material-ui/core";
+import { Backspace, LocalPhone } from "@material-ui/icons";
 import { Header } from "components/Header";
 import { labels } from "labels";
 
+interface Options {
+  value: string;
+  label: string;
+}
+interface SettingsRadioGroupProps {
+  formLabel: string;
+  name: string;
+  options: Options[];
+  value: string;
+  setValue: (to: string) => void;
+}
+interface SettingKeyPadProps {
+  buttonsValues: string[];
+  number: string;
+  setNumber: (newNum: any) => void;
+}
+const SettingsRadioGroup: React.FC<SettingsRadioGroupProps> = ({
+  name,
+  formLabel,
+  options,
+  value,
+  setValue,
+}) => {
+  return (
+    <FormControl component="fieldset">
+      <FormLabel component="legend">{formLabel}</FormLabel>
+      <RadioGroup
+        name={name}
+        value={value}
+        onChange={(e) => setValue(e.target.value)}
+      >
+        {options.map((option, index) => (
+          <FormControlLabel
+            key={index}
+            value={option.value}
+            control={<Radio color="primary" />}
+            label={option.label}
+          />
+        ))}
+      </RadioGroup>
+    </FormControl>
+  );
+};
+
+const SettingKeyPad: React.FC<SettingKeyPadProps> = ({
+  buttonsValues,
+  number,
+  setNumber,
+}) => {
+  return (
+    <Grid container spacing={0}>
+      <Grid item xs={12}>
+        <Input
+          value={number}
+          onChange={(e) => setNumber(e.target.value)}
+          endAdornment={
+            <InputAdornment position="end">
+              <IconButton
+                aria-label="delete"
+                onClick={() => setNumber(number.slice(0, -1))}
+              >
+                <Backspace fontSize="small" />
+              </IconButton>
+            </InputAdornment>
+          }
+        />
+      </Grid>
+      {buttonsValues.map((button, index) => (
+        <Grid item xs={4} key={index}>
+          <Button onClick={() => setNumber(`${number}${button}`)}>
+            {button}
+          </Button>
+        </Grid>
+      ))}
+      <Grid item xs={12}>
+        <IconButton aria-label="call">
+          <LocalPhone />
+        </IconButton>
+      </Grid>
+    </Grid>
+  );
+};
+
 export const SettingsPage: React.FC = () => {
-  return <Header title={labels.telephonySettings} toPage="about" />;
+  const [currentPhone, setCurrentPhone] = useState("computer");
+  const [currentPhoneOptions] = useState([
+    {
+      value: "computer",
+      label: `${labels.computerVoIP}`,
+    },
+    {
+      value: "officePhone",
+      label: `${labels.officePhone}`,
+    },
+    {
+      value: "otherPhone",
+      label: `${labels.otherPhone}`,
+    },
+  ]);
+  const [forwardYourCalls, setForwardYourCalls] = useState(
+    "cancelCallForwarding"
+  );
+  const [forwardYourCallsOptions] = useState([
+    {
+      value: "cancelCallForwarding",
+      label: `${labels.dontForward}`,
+    },
+    {
+      value: "forwardingToVoicemail",
+      label: "Forwarding to voicemail",
```

Comment:
This is not localized . You can use the `forwarding to` label and the `Voicemail` label to localize this string